pymess/models/dialer.py

```diff
@@ -77,6 +77,9 @@ class Meta:
 
 
 class DialerMessage(AbstractDialerMessage):
+
+    is_done = models.BooleanField(verbose_name=_('is done'), null=False, default=False)
```

Comment:
Hodil by se popis co to vlastně znamená, bohužel dokumentace pro celou část dialer chybí :(

Reply:
bol som v tom, ze toto je dokumentacia `docs/dialer.rst` tam toto pole doplnim...